<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secret Keeper Router</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="gradient">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>680</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/gradient.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="group_creation_button">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>19</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rename_users_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_from_group_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rename_group_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="add_to_group_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_1_1">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>90</y>
      <width>285</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>Добавить пользов.</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_1_2">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>140</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>в группу</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_2_1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>Удалить пользов.</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_2_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>из группы</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_3_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>Переименовать </string>
    </property>
   </widget>
   <widget class="QLabel" name="text_4_1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>Переименовать</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_3_2">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>400</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>пользов.</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_4_2">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>400</y>
      <width>102</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>группу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_group_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>660</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string>Удалить группу</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
